test/mpi/.gitignore

```diff
@@ -386,38 +412,46 @@
 /datatype/vecblklen
 /datatype/zeroblks
 /datatype/zeroparms
+/dtpools/aclocal.m4
+/dtpools/autom4te.cache/requests
+/dtpools/confdb/
+/dtpools/confdb/ar-lib
+/dtpools/confdb/compile
+/dtpools/confdb/depcomp
+/dtpools/confdb/install-sh
+/dtpools/confdb/missing
+/dtpools/libtool
+/dtpools/stamp-h1
 /errhan/adderr
 /errhan/commcall
+/errhan/commcall_oldapi
 /errhan/dynamic_errcode_predefined_errclass
-/errhan/errcode
 /errhan/errcode.c
 /errhan/errfatal
-/errhan/errring
-/errhan/errstring
```

Comment:
The removed entries here are listed as `EXTRA_PROGRAMS` in `Makefile.am`. I think they are valid, though they are not built by `make all`.

Reply:
Ok I'll add them back. That's just the /errhan/ entries?